components/automatic-points-table/commons/automatic-points-table.lua

```diff
@@ -49,30 +50,31 @@ function AutomaticPointsTable.run(frame)
 	-- mw.logObject(pointsTable.parsedInput.tournaments)
 	-- mw.logObject(pointsTable.parsedInput.teams)
 	-- mw.logObject(tournamentsWithPlacements)
-	mw.logObject(sortedDataWithPositions)
-
-	return pointsTable:generatePointsTable(sortedDataWithPositions, tournamentsWithResults)
+	-- mw.logObject(sortedDataWithPositions)
+	local positionBackgrounds = pointsTable.parsedInput.positionBackgrounds
+	return pointsTable:makePointsTable(sortedDataWithPositions, tournamentsWithResults,
+		positionBackgrounds)
 end
 
 function AutomaticPointsTable:parseInput(args)
-	local pbg = self:parsePositionBackgroundData(args)
+	local positionBackgrounds = self:parsePositionBackgroundData(args)
 	local tournaments = self:parseTournaments(args)
 	local teams = self:parseTeams(args, #tournaments)
 	return {
-		pbg = pbg,
+		positionBackgrounds = positionBackgrounds,
 		tournaments = tournaments,
 		teams = teams
 	}
 end
 
---- parses the pbg arguments, these are the background colors of specific positions
---- Usually used to indicate where a team in a specific position will end up qualifying to
+--- parses the position_background arguments, these are the background colors of specific
+--- positions, usually used to indicate if a team in a specific position will end up qualifying
 function AutomaticPointsTable:parsePositionBackgroundData(args)
-	local pbg = {}
-	for _, background in Table.iter.pairsByPrefix(args, 'pbg') do
-		table.insert(pbg, background)
+	local positionBackgrounds = {}
+	for _, background in Table.iter.pairsByPrefix(args, 'position_background') do

```

Comment:
Wiki code by convention is no space (or underscores) and in lower case only.
Not use if the wikicode should change to this, bit too long. I agree that pbg is not very clear. maybe just `positionbg`?

Reply:
Seems reasonable, will chnage it